fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -1389,4 +1399,12 @@ public void logRemoveAnalyzeJob(AnalyzeJob job) {
     public void logModifyTableColumn(ModifyTableColumnOperationLog log) {
         logEdit(OperationType.OP_MODIFY_HIVE_TABLE_COLUMN, log);
     }
+
+    public void logCreateCatalog(CreateExternalCatalogOperationLog log) {
+        logEdit(OperationType.OP_CREATE_EXTERNAL_CATALOG, log);
```

Comment:
should add this later.
Because we may change the log content when implementing. Better log it later.

Reply:
> implementing

I will split this pr to another 2 later, catlog persistent need others help review.